chore: reporting updates and fixes

brownoxford opened the pull request:
* Update size of boxes in facet report to prevent overlapping
* Color pass/fail based on number of assertions passed or failed
* Clean up tooltips on facet report

FIX: #278

OR13 commented on reporting/postman_reporter/report_config.py:
```diff
@@ -51,6 +51,11 @@
+    "(?)": "darkgoldenrod",
+    "Pass": "darkgreen",
+    "Fail": "darkred",
+    "Partial": "darkgoldenrod",
```
hmm.... not sure about adding this.

OR13 replied:
Can you explain why its valuable?

brownoxford replied:
It serves to show when a test is partially passing, e.g., when a test case contains multiple assertions and only some of them fail. The other option would be to simply show those as failures.

If you would like to add a change request to modify `darkgoldenrod` to `darkred` it will produce the same output as before.

OR13 replied:
i see, I am ok with that, as long as we agree with respect to what "passing" means.

We should define passing as:

"All green, no yellow, and no red".

Anything other than "passing" is failing.

We are not planning on allowing for folks to say "we don't want to demonstrate interop on that feature, so its ok that we are "yelllow" for it...".

OR13 reviewed (changes requested):
Unsure about adding "partial" to the test report. 

Need additional details to approve / deny.

defaulting to deny.

mprorock commented:
> Unsure about adding "partial" to the test report.
> 
> Need additional details to approve / deny.
> 
> defaulting to deny.

I definitely lean this way as well

OR13 reviewed (changes requested):
change "Partial" to "Fail-partial", colors are good to keep as is.

mprorock commented on reporting/postman_reporter/report_charts.py:
```diff
@@ -181,13 +181,36 @@ def getCards(df):
+
+    # Map number of tests and passing assertions to a status.
+    df.loc[df["passed"] == df["tests"], ["status"]] = "Pass"
+    df.loc[df["passed"] < df["tests"], ["status"]] = "Partial"
```
```suggestion
    df.loc[df["passed"] < df["tests"], ["status"]] = "Fail (Partial)"
```

mprorock commented on reporting/postman_reporter/report_charts.py:
```diff
@@ -225,35 +248,61 @@ def getFacet(df):
+
+    # Map number of tests and passing assertions to a status.
+    df.loc[df["passed"] == df["tests"], ["status"]] = "Pass"
+    df.loc[df["passed"] < df["tests"], ["status"]] = "Partial"
```
```suggestion
    df.loc[df["passed"] < df["tests"], ["status"]] = "Fail (Partial)"
```

mprorock commented on reporting/postman_reporter/report_charts.py:
```diff
@@ -225,35 +248,61 @@ def getFacet(df):
+
+    # Map number of tests and passing assertions to a status.
+    df.loc[df["passed"] == df["tests"], ["status"]] = "Pass"
+    df.loc[df["passed"] < df["tests"], ["status"]] = "Partial"
```
```suggestion
    df.loc[df["passed"] < df["tests"], ["status"]] = "Fail (Partial)"
```

mprorock reviewed (changes requested):
see minor adjustments I suggested @brownoxford @OR13

brownoxford commented:
I modified the text used in the status column for partial failures to be `Fail (Partial)` and fixed a column name typo in `getFacet()`.

We can also play around with the text in the hover popup, e.g., stop showing `shape` and `size`, start showing `Passing: x/y`, etc.

brownoxford commented:
Here are some screenshots that show how reports will now look:

![Screen Shot 2022-07-19 at 12 47 03 PM](https://user-images.githubusercontent.com/755209/179805768-9ada2011-1f66-4a39-b671-d21ed6c3a53f.png)
![Screen Shot 2022-07-19 at 12 47 30 PM](https://user-images.githubusercontent.com/755209/179805816-a3028d8a-4b86-4abb-97bc-7f11d00dd31e.png)
![Screen Shot 2022-07-19 at 12 47 38 PM](https://user-images.githubusercontent.com/755209/179805858-6a1c5997-c014-407d-850b-671a1a624c4a.png)
